docs/guide/deployment/authorization-server.md

```diff
@@ -5,36 +5,52 @@
 ```bash
 docker pull plgd/authorization:latest
 ```
-Or by using source
-```bash
-# Dowonload github source
-git clone https://github.com/plgd-dev/cloud.git 
-
-# Build the source
-cd cloud/ 
-make build
-```
 
 ## Docker Run
```

Comment:
Hm Trishia, but what is the reasoning behind starting the authorization server alone?

Reply:
@ondrejtomcik 
What is the meaning you mentioned above? Describe more detail you want plz.